cmd/sqlflowserver/main_test.go

```diff
@@ -875,6 +879,19 @@ INTO sqlflow_models.my_dnn_model_custom_functional;`
 	}
 }
 
+func CaseTrainWithCSVLabel(t *testing.T) {
```

Comment:
I am a little confused -- why is the label called CSV label? CSV is a file format, right? Should we say `concat_label`?

Reply:
CSV stands for "Comma Separated Values", since it's some values, I think it's OK to represent the data in the database cell.

Reply:
I see. Then `CSVLabel` means `comma-separated value label`, which is ambiguous -- is it a value or a label.  Should we use the name `CaseTrainWithCommaSeparatedLabel`?

If there could be column-separated label and semi-column-separated labels in the future, we might want `ConcatenatedLabel` or `ConcatLabel` instead.

Reply:
Sounds good, will update the naming in next PR.